resources/data/detailed_rates/Flat/Sample Flat Rate.json

```diff
@@ -0,0 +1,647 @@
+{
+  "items": [
+    {
+      "name": "Sample Tiered Rate",
+      "sector": "Residential",
+      "description": "Sample file defined by BEopt.",
+      "fixedchargeunits": "$/month",
+      "fixedchargefirstmeter": 8,
+      "energyweekdayschedule": [
```

Comment:
@yingli-NREL For some context on the use of tariff JSON files containing detailed utility rate structures, see the "Detailed" section of the [OS-HPXML docs](https://openstudio-hpxml.readthedocs.io/en/latest/workflow_inputs.html#electricity-rates). Basically, the format of these files are prescribed by OpenEI API version 7.

Regarding your specific question, there are 12 lists so that differences in rates from month to month can be supported.

Reply:
Got it. 12 lists mean the rates in 12 months.